### DISCUSSION TITLE
为什么视频不能播放，图片和音频都是正常可以访问的

### DISCUSSION THREAD

**Discussion by hsuromky** [AlistGo/alist#2178] [General]
奇了怪了，alist部署在家里电脑，不管是网页访问还是用支持webdav的app访问，图片和音频都是可以正常打开的，就是视频不能播放，用nplayer提示此文件不可播放：403  用reex提示loading failed。。。

**Comment by hsuromky** [comment]
而且alist网页直接播放提示播放器左上角显示“重新连接12345,加载失败”

**Reply by Nicobalice** [reply]
昨晚我也遇到了 不知道咋解决

**Comment by zepang** [comment]
同样遇到这个问题

**Reply by hsuromky** [reply]
阿里云盘的刷新令牌重新获取了一下就好了，你可以试试。可是百度云盘的还是不行，而且查看2m以上大小的图片就提示载入失败，2m以下大小是可以正常打开的

**Reply by zepang** [reply]
令牌重新获取了，还是一样

**Reply by zepang** [reply]
![Uploading image.png…]()
重试好几次了，还是一样

**Reply by zepang** [reply]
百度云盘的我目前是音频可以，视频和阿里云一样，重试几次之后，直接就放不了

**Reply by hsuromky** [reply]
百度云盘改成本地代理试试，我改过之后有几个可以播放有几个不能，很奇怪

**Reply by zepang** [reply]
还是不行，我是部署在服务器上的

**Reply by megumi335400** [reply]
+1，音频和图片都可以就是视频放不出来

**Comment by zgqzgqzc** [comment]
我也同样遇到这个问题 一直没有解决

**Comment by se7enmuting** [comment]
nplayer 可以看到视频文件，但就是播放不了？是不是用了 https 了？反代的设置里请不要开启 HTTP/2 ，目前 nplayerr(3.12.17) 不支持 HTTP/2，关了用 HTTP/1.1 就好了。

**Comment by Nephino** [comment]
考虑是不是 config.json 内的 `site_url` 选项在末尾添加了额外的 "/" ，参考[官方文档](https://alist.nn.ci/zh/config/configuration.html#site-url)。这是我遇到这个问题的原因，修复就可以访问了。

**Comment by daihaosh** [comment]
我也遇到这个问题了，在“存储”里面“启用签名”就可以播放了。

**Comment by RectEgg** [comment]
> 奇了怪了，alist部署在家里电脑，不管是网页访问还是用支持webdav的app访问，图片和音频都是可以正常打开的，就是视频不能播放，用nplayer提示此文件不可播放：403 用reex提示loading failed。。。

解决了嘛？
